pkg/util/kube/storage.go

```diff
@@ -321,3 +321,11 @@ func isEncryptedStorageClass(
 
 	return false, "", nil
 }
+
+func GetPVC(ctx context.Context, k8sClient ctrlclient.Client, pvcNamespace string, pvcName string) (*corev1.PersistentVolumeClaim, error) {
```

Comment:
* Exported functions should have Go documentation.
* Also, where is the unit test for this to provide coverage?
* Please limit things to 80 columns.
* The `pvcNamespace` and `pvcName` do not need the `pvc` prefix since the function is called `GetPVC` 😁

Reply:
Dropped what? I do not see any fixes here. It's the same width, 140 chars, with the same argument names, I do not see that anything was done to this. Please update it according to my notes above. That or drop the function since it does not do anything like @bryanv said. Which means, you know, removing this file.